### ISSUE TITLE
docs for 3.2

### ISSUE THREAD

**Issue by zeffii** [zeffii/mesh_tiny_cad#80]
from memory, as I have little free time. i'll try to get back to this soon.

```rst
mesh tiny cad
=========

This tool offers a few operations normally found in CAD programs, like; Intersection, Extend, finding the Center of a circle, Bisect and Perpendicular. Below is a description of individual operations.

VTX
----
**Vertex welded to intersection of two edges**

V2X
---
**Add Vertex at the intersection of two edges**

XALL
----
**Intersect all selected edges**
This welds a vertex at the intersection of any edges present in the selection.

CCEN
----
**Finds the geometric center of a circle from 3 vertices**
This tool will offer to regenerate a new Circle at the origin and orientation of the found circle, and let you decide the resolution of the circle.
```

```

**Comment by Blendify** [comment]
@zeffii with your permission can I copy/use this content to build an official manual page for you?

Ill give you commit access to if you want to update it in the future but it seems for the most part you have abandoned this project.

**Comment by zeffii** [comment]
yes that's fine.. but let me fill it out a little further before committing. + some pictures from the http://zeffii.github.io/mesh_tiny_cad/ docs.

**Comment by zeffii** [comment]
but @Blendify if I don't return to this within a time you find reasonable, you have my go-ahead to do with it what you see fit.  sadly i have different priorities at the moment.

**Comment by zeffii** [comment]
![bix](https://user-images.githubusercontent.com/619340/156788894-cb1aca06-c8ac-40d3-8670-9f07790e8de7.png)
![ccen](https://user-images.githubusercontent.com/619340/156788898-0d9449f3-98aa-4a82-839a-35123200a9a7.png)
![e2f](https://user-images.githubusercontent.com/619340/156788900-02ec40c7-f109-44fc-b4e9-fae54d1281f1.png)
![v2x](https://user-images.githubusercontent.com/619340/156788901-4bc947f1-f648-45e8-9d54-175f5a28077b.png)
![vtx](https://user-images.githubusercontent.com/619340/156788904-0bd97ed4-00c0-4794-a9af-b3f9278b1c2d.png)
![xall](https://user-images.githubusercontent.com/619340/156788906-4743cdb4-580c-44f6-85fd-86fa4f49ca67.png)
